kairon/importer/validator/file_validator.py

```diff
@@ -280,11 +314,17 @@ def verify_utterances_in_stories(self, raise_exception: bool = True):
         utterance_actions = utterance_actions.difference(form_utterances)
 
         for utterance in utterance_actions:
-            if utterance not in stories_utterances and utterance not in system_triggered_actions.union(fallback_action):
-                msg = f"The utterance '{utterance}' is not used in any story."
-                if raise_exception:
-                    raise AppException(msg)
-                utterance_mismatch_summary.append(msg)
+            msg = f"The utterance '{utterance}' is not used in any story."
+            if multiflow_stories_utterances:
```

Comment:
simplify this
use sets and join utterances from multiflow as well as normal stories